controller/controller_server/csi_controller_server.py

```diff
@@ -106,8 +103,8 @@ def CreateVolume(self, request, context):  # pylint: disable=too-many-branches
                     logger.debug(
                         "volume was not found. creating a new volume with parameters: {0}".format(request.parameters))
 
-                    array_mediator.validate_supported_capabilities(capabilities)
-                    volume = array_mediator.create_volume(volume_final_name, size, capabilities, pool)
+                    array_mediator.validate_supported_capabilities(space_efficiency_type)
```

Comment:
then function name should be aligned with its parameter name

Reply:
done